app/helpers/react_on_rails_helper.rb

```diff
@@ -34,6 +34,9 @@ module ReactOnRailsHelper
   #                    true, you'll still see the errors on the server.
   #    raise_on_prerender_error: <true/false> Default to false. True will raise exception on server
   #       if the JS code throws
+  #    skip_display_none: <true/false> Default to false. True will not add `style: "display:none"`
```

Comment:
This doesn't need to be configurable at the component level. I.e. Let's not put this in the view helper.